packages/big-design/src/components/Table/__snapshots__/spec.tsx.snap

```diff
@@ -565,7 +565,7 @@ exports[`renders a simple table 1`] = `
   id="bd-table-1"
 >
   <thead
-    class="styled__StyledTableHead-sc-1p8j1x-0"
+    class=""
```

Comment:
I believe this is related to https://github.com/styled-components/jest-styled-components/issues/395